lib/std/coff.zig

```diff
@@ -162,20 +163,51 @@ pub const Coff = struct {
         try self.loadOptionalHeader();
     }
 
+    fn readStringFromTable(self: *Coff, offset: u32, buff: []u8) !u32 {
+        if (self.coff_header.pointer_to_symbol_table == 0) {
+            // No symbol table therefore no string table
+            return error.MissingStringTable;
+        }
+        const string_table_offset = self.coff_header.pointer_to_symbol_table + (self.coff_header.number_of_symbols * 18) + offset;
+        const in = self.in_file.reader();
+        const old_pos = try self.in_file.getPos();
+
+        try self.in_file.seekTo(string_table_offset);
+
+        var i: u8 = 0;
+        while (i < buff.len) : (i+=1) {
```

Comment:
This loop can be simplified down to:
```zig
const str = try in.readUntilDelimiterOrEof(buf, 0)
```

Reply:
Good to know. I'm still getting used to the stdlib